### PR TITLE
Update nl.po

### PR DESCRIPTION
**Pull request by gvmelle**
I translated the obvious and corrected a lot of bullshit entries. Don't know what to do with those code-texts like activeCaption. Why are those in a po-file???? Seems not relevant for a user interface, but more for developers and coders.

### CODE REVIEW

**Comment by vadz** [comment]
Was this closed on purpose?

As for "active caption" and such, they seem to be coming from `wxPropertyGrid` sources. I'm not sure if they need to be translated but I can't totally exclude it neither.

**Comment by gvmelle** [comment]
No, not in purpose. Could you open it and merge it?

Verstuurd vanaf mijn iPhone

File: locale/nl.po [hunk 1]
```diff
@@ -1490,15 +1487,15 @@ msgstr "Bitmap renderer kan geen waarde renderen; waarde type: "
...
 
 #: ../src/propgrid/advprops.cpp:1705
 msgid "Blank"
-msgstr ""
+msgstr "Blanco"
 
 #: ../src/propgrid/advprops.cpp:1553
 msgid "Blue"
-msgstr ""
+msgstr "blauw"
```
Comment (discnl) [inline locale/nl.po]: This should be "Blauw".

File: locale/nl.po [hunk 2]
```diff
@@ -3425,9 +3413,9 @@ msgid "Failed to load shared library '%s'"
 msgstr "Laden van gedeelde bibliotheek '%s' mislukt"
 
 #: ../src/osx/core/sound.cpp:118
-#, fuzzy, c-format
+#, c-format
 msgid "Failed to load sound from \"%s\" (error %d)."
-msgstr "Laden van bronbestand \"%s\" mislukt."
+msgstr "Laden geluid van \"%s\" mislukt (fout %d)."
```
Comment (discnl) [inline locale/nl.po]: Personally I'd prefer "Laden van geluid \"%s\" [...]" or "Laden geluid vanaf \"%s\" [...]" . It sounds a little less 'krom' IMHO.

File: locale/nl.po [hunk 3]
```diff
@@ -8028,21 +7980,18 @@ msgstr "Windows/DOS OEM Cyrillic (CP 866)"
 
 #. TRANSLATORS: Name of keyboard key
 #: ../src/common/accelcmn.cpp:109
-#, fuzzy
 msgid "Windows_Left"
-msgstr "Windows 7"
+msgstr "Windows_Left"
```
Comment (discnl) [inline locale/nl.po]: I don't think all of the keys need translating (though I suppose people that exclusively use Dutch on their systems would disagree) but maybe at least this one, and Windows_Right, could be? In this case "Linker Windows-toets".

File: locale/nl.po (hunk 2 above)
Comment (gvmelle) [inline locale/nl.po]: You are free to correct things. I wonder however where in a user interface one sees things like windows_left or activeCaption. It looks like variables or keywords to me. 

Ciao,

Gideon

~~o
<#\,
()/()

(Verstuurd vanaf mijn iPad air)

**Comment by gvmelle** [comment]
By the way, how come I don't see myself as translator? What is the relation with the old po-edit translation website http://www.wxwidgets.org/about/translations/?info=nl ?
There I see myself, as expected, as translator. Is Vadim Zeilin still in charge?

**Comment by vadz** [comment]
Sorry, I don't understand these questions. You (@gvmelle) are mentioned as translator on the page you linked (just search for your name). And that page has nothing to do with Poedit, it just shows the latest translations status, so what relationship is it supposed to have with anything? Finally, I'm as much "in charge" as I ever was...

**Comment by gvmelle** [comment]
I meant I did not see my name on the github site as translator. Only on the old site. And also I meant po-files and not poedit files. ( I happened to use poedit to translate them). And apparently VZ stands for Vadim Zeitlin. Sorry! I did not get that. 

Anyway you can count me out as translator, you apparently have enough new qualified ones. 

Verstuurd vanaf mijn iPhone

**Comment by discnl** [comment]
Committed as c41aaf2, thanks! Also I'm not aware of any active Dutch translators and even if there are you're always welcome to translate of course.

**Comment by tierra** [comment]
Uh... the list of Dutch translators on the website is exactly the same as it was on the old website (taken from the source code of both sites):

```
Dutch Translators on Old site:

   'nl' => '<a href="mailto:phubers@@solve-i-t.com">Patrick Hubers</a>,<br/>
            <a href="mailto:gvmelle@@gmail.com">Gideon van Melle</a>,<br/>
            <a href="mailto:thomasderocker@@hotmail.com">Thomas De Rocker</a>',

Dutch Translators on New site:

* Dutch (nl)
  * Patrick Hubers <<phubers@solve-i-t.com>>
  * Gideon van Melle <<gvmelle@gmail.com>>
  * Thomas De Rocker <<thomasderocker@hotmail.com>>
```

In any case, you're more than welcome to request to be added if you want. We're just not in the habit of always publishing everyone's email address online without their permission, so maybe you never asked?

**Comment by gvmelle** [comment]
Just not aware of the change from the old site to the Github site.
And then, after finding out, wondering why the translators just not also
got "moved" from the old to the new.
I remember getting warnings when an update needed additional strings to be
translated, that's why a was not in the habit to frequently checking.

On Thursday, May 14, 2015, Bryan Petty <notifications@github.com> wrote:

> Uh... the list of Dutch translators on the website is exactly the same as
> it was on the old website (taken from the source code of both sites):
>
> Dutch Translators on Old site:
>
>    'nl' => '<a href="mailto:phubers@ <javascript:_e(%7B%7D,'cvml','phubers@');>@solve-i-t.com">Patrick Hubers</a>,<br/>
>             <a href="mailto:gvmelle@ <javascript:_e(%7B%7D,'cvml','gvmelle@');>@gmail.com">Gideon van Melle</a>,<br/>
>             <a href="mailto:thomasderocker@ <javascript:_e(%7B%7D,'cvml','thomasderocker@');>@hotmail.com">Thomas De Rocker</a>',
>
> Dutch Translators on New site:
>
> * Dutch (nl)
>   * Patrick Hubers <<phubers@solve-i-t.com <javascript:_e(%7B%7D,'cvml','phubers@solve-i-t.com');>>>
>   * Gideon van Melle <<gvmelle@gmail.com <javascript:_e(%7B%7D,'cvml','gvmelle@gmail.com');>>>
>   * Thomas De Rocker <<thomasderocker@hotmail.com <javascript:_e(%7B%7D,'cvml','thomasderocker@hotmail.com');>>>
>
> In any case, you're more than welcome to request to be added if you want.
> We're just not in the habit of always publishing everyone's email address
> online without their permission, so maybe you never asked?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wxWidgets/wxWidgets/pull/25#issuecomment-102109243>.
>


-- 
Ciao,

Gideon

~~o
<#\,
()/()

**Pull request closed without merging**